.github/workflows/tests.yml

```diff
@@ -47,11 +47,33 @@ jobs:
             ~/.cargo/registry/cache/
             ~/.cargo/git/db/
             target/
-          key: ${{ runner.os }}-cargo-clippy-${{ hashFiles('**/Cargo.lock') }}
+          key: ${{ runner.os }}-sqlx-prepare-${{ hashFiles('**/Cargo.lock') }}
       - name: Install sqlx
         run: cargo install sqlx-cli --no-default-features --features postgres
       - name: Run the test sqlx migrations
         run: cargo sqlx migrate run
+      - name: Check that the sqlx prepared query metadata is up-to-date
+        run: cargo sqlx prepare --workspace --check -- --all-targets --all-features
```

Comment:
is all targets/features needed here when we have specified a list of features in sqlx dep?